packages/svelte/vitest.config.js

```diff
@@ -22,6 +22,7 @@ export default defineConfig({
 	test: {
 		dir: 'test',
 		reporters: ['dot'],
+		include: ['**/*.test.js'],
```

Comment:
why did this line need to change? there's no `.test.js` file added in this PR

Reply:
I think I committed it to the wrong branch. Removed.